doc/reference/reference_lua/box_index/parts.rst

```diff
@@ -8,7 +8,15 @@ index_object:parts
 
     .. data:: parts
 
-        The index's key parts.
+    The index's key parts. 
+    Since version :doc:`3.0.0 </release/3.0.0>`, the ``index_object:parts`` can operate methods
+    :ref:`extract_key() <key_def-extract_key>`,
+    :ref:`compare() <key_def-compare>`,
+    :ref:`compare_with_key() <key_def-compare_with_key>`,
+    :ref:`merge() <key_def-merge>`,
+    :ref:`totable() <key_def-totable>`.
```

Comment:
`totable()` is absent in `index_object:parts`, because it doesn't make sense (`index_object:parts` is already a table).